src/calculator.js

```diff
@@ -1,47 +1,31 @@
 exports._check = () => {
   // DRY up the codebase with this function
-  // First, move the duplicate error checking code here
-  // Then, invoke this function inside each of the others
-  // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
     throw new TypeError(`${x} is not a number`);
   }
   if (typeof y !== 'number') {
     throw new TypeError(`${y} is not a number`);
```

Comment:
&#39;y&#39; is not defined.

###### See http://eslint.org/docs/rules/no-undef for details about this rule